src/components/util/validators.js

```diff
@@ -74,4 +74,17 @@ export const validatePostalCode = errorMessage => (rule, value, callback) => {
     callback(errorMessage);
   }
   callback();
+};
+
+/**
+ * Custom Logo URL validator
+ * @param errorMessage
+ * @returns {Function}
+ */
+export const validateImageUrl = errorMessage => (rule, value, callback) => {
+  // If value is not empty and not a valid URL or does not have an appropriate extension it will be marked as invalid
+  if (value && (!isURL(value) || value.match(/\.(jpeg|jpg|gif|png)$/) === null)) {
```

Comment:
Nice idea, but I imagine it won't be enough I'm afraid. At least for other pictures they are often missing ext. And logos could be svgs. But let us see how it works. It might be that logos behave more nicely.